firebase-appdistribution/src/main/res/values/strings.xml

```diff
@@ -13,40 +13,37 @@
 <!-- limitations under the License. -->
 
 <resources>
-  <string name="signin_dialog_title">Enable testing features</string>
-  <string name="singin_dialog_message">Enable testing features like new build alerts and in-app feedback.</string>
-  <string name="singin_yes_button">Turn on</string>
-  <string name="singin_no_button">Not now</string>
```

Comment:
Would "Cancel" be more appropriate here, if it's canceling an action?

Reply:
The action (trying to enable the SDK) was likely not taken by the user in this case. It is typically triggered by the developer when the app is first opened.